### PR TITLE
Add New Case Study to Resources Page

### LINKED ISSUE #826: Add New Case Study to Resources Page
- **URL**:
- **Browser version**:
- **Operating system**:

We have a new Node.js Foundation Case Study that we'd like added below the Uber Case Study on https://nodejs.org/en/foundation/case-studies/

Title: Node.js Helps Fusion Marketing Build Memorable Digital Experiences

Description: Fusion Marketing is a St. Louis, MO-based event marketing firm that specializes in creating custom interactive experiences. Node.js Everywhere has powered Fusion's digital expansion from developing app APIs to a 360 degree physical interactive group. 
Read the full case study: (please insert link to PDF in "full case study")

[Node_CaseStudy_Fusion_Final.pdf](https://github.com/nodejs/nodejs.org/files/366472/Node_CaseStudy_Fusion_Final.pdf)

### PR DESCRIPTION
**Pull request by stevemao**
Fixes #826

### CODE REVIEW

File: locale/en/foundation/case-studies.md [hunk 1]
```diff
@@ -5,23 +5,28 @@ layout: foundation.hbs
...
\ No newline at end of file
+The ride-sharing platform Uber has an ambitious goal: to make transportation
+as reliable as running water, everywhere and for everyone. Learn why Uber
+chose Node.js to power its platform and massive-matching system.
+
+Read the [full case study](/static/documents/casestudies/Nodejs-at-Uber.pdf).
+
+## Node.js Helps Fusion Marketing Build Memorable Digital Experiences
+
+Fusion Marketing is a St. Louis, MO-based event marketing firm that specializes in creating custom interactive experiences. Node.js Everywhere has powered Fusion's digital expansion from developing app APIs to a 360 degree physical interactive group.
```
Comment (lpinca) [inline locale/en/foundation/case-studies.md]: Keep the limit of 80 chars per line? Mostly for consistency with the rest of the document.

File: locale/en/foundation/case-studies.md [hunk 2]
```diff
@@ -5,23 +5,28 @@ layout: foundation.hbs
...
+Read the [full case study](/static/documents/casestudies/Node_CaseStudy_Fusion_Final.pdf)
```
Comment (lpinca) [inline locale/en/foundation/case-studies.md]: Add a new line before this? Again for consistency with the above case.

**Comment by lpinca** [comment]
LGMT with a couple of nits.

**Comment by stevemao** [comment]
@lpinca Please take a look again.

**Comment by lpinca** [comment]
👍

**Comment by gtewallace** [comment]
Thanks very much all!

**Pull request merged**